src/addons/addonapi.h

```diff
@@ -27,6 +28,9 @@ class AddonApi final : public QQmlPropertyMap {
                                  const QJSValue& callback);
   Q_INVOKABLE void log(const QString& message);
 
+  Q_INVOKABLE void setTimeout(int interval, const QJSValue& callback);
```

Comment:
Consider if a name like scheduleCallback or setTimedCallback are better names, and change clearTimers accordingly.